admin/base/settings/defaults.py

```diff
@@ -20,6 +20,15 @@
 DEBUG = osf_settings.DEBUG_MODE
 DEBUG_PROPAGATE_EXCEPTIONS = True
 
+# secure mode for using https in local development
+SECURE = osf_settings.SECURE_MODE
+LOCAL = osf_settings.LOCAL_MODE
+
+# session:
+SESSION_COOKIE_HTTPONLY = True
```

Comment:
Agree. 
\[FYI\] Django's default value for `SESSION_COOKIE_HTTPONLY` is true. It is not an issue. But we still want to make the two consistent.